add-on/TA-Demisto/default/app.conf

```diff
@@ -7,9 +7,12 @@ build = 2
 
 [launcher]
 author = Palo Alto Networks
-version = 4.1.2
+version = 4.2.0
 description = This application provides an alert action to create an incident in Cortex XSOAR.
 
+[id]
+version = 4.2.0
```

Comment:
let's update documentation that this needs to be bumped in every new version update (right?)